app/javascript/packs/components/DropdownOption.jsx

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 
 class DropdownOption extends React.Component {
   componentDidMount() {
-    const selected_option_id = this.props.answer.selected_option_id;
-    if (selected_option_id) {
-      this.props.onSelect(selected_option_id);
+    if (this.props.answer) {
+      const selected_option_id = this.props.answer.selected_option_id;
```

Comment:
Cool trick;

`const { selected_option_id: selected } = this.props.answer;`

❓